src/EventStore.Core/TransactionLog/Scavenging/Data/DiscardDecision.cs

```diff
@@ -7,5 +7,6 @@ public enum DiscardDecision {
 		// index will just keep such indexentries.
 		MaybeDiscard,
 		Keep,
+		DiscardIfAllPreviousDiscarded,
```

Comment:
What do you think about calling this `PreviouslyDiscarded`

Reply:
yup, that sounds better :+1: i had previously used `Discarded` but since it wasn't a 'decision' about what to do with the event, i'd renamed it to: `DiscardIfAllPreviousDiscarded` but i think it's ok to have an exception for this one.

Reply:
following our discussion, i've renamed it to `AlreadyDiscarded`